proxy/envoy/config.go

```diff
@@ -92,6 +89,13 @@ func buildConfig(listeners Listeners, clusters Clusters, lds bool, mesh *proxyco
 		StatsdUDPIPAddress: mesh.StatsdUdpAddress,
 	}
 
+	if mesh.ProxyAdminPort > 0 {
+		out.Admin = &Admin{
+			AccessLogPath: DefaultAccessLog,
+			Address:       fmt.Sprintf("tcp://%s:%d", WildcardAddress, mesh.ProxyAdminPort),
```

Comment:
That might be a good idea. Let's bind to 127.0.0.1 by default for now (that's more secure to begin with).